src/lib/AccountIdentifier.ts

```diff
@@ -0,0 +1,36 @@
+import { CLPublicKey } from './CLValue';
+
+export interface AccountIdentifier {
+  toJSON(): string;
+}
+
+export class PublicKey implements AccountIdentifier {
+  private key: string;
+
+  constructor(key: string) {
+    this.key = key;
+  }
+
+  toJSON(): string {
+    return this.key;
+  }
+}
+
+export class AccountHash implements AccountIdentifier {
```

Comment:
Why don't we use existing classes for these?

Reply:
Could you point me to the class that is a polymorphic representation of either PublicKey or AccountHash? I didn't find it.

Reply:
The `state_get_account_info` RPC method doesn't take a `CLValue`. It takes an `AccountIdentifier`.
So we still need something that knows how to make a hex-string like "017b14afdcaa0c2890bc317352aff880e3d5d4d703500e4d2415de93daaa43b64e" from a CLPublicKey or "account-hash-73911a773cd124ae43572066dedb321e71d93dd18c337ef1365c8bc19e95895f" from CLAccountHash

Reply:
Ok, after some thought I just changed the type of accountIdentifier to CLPublicKey, since it has `toAccountHashStr`. Our API clients don't have to implement every possible alternative that arguments can be constructed to the json API

Reply:
Thanks for updates, but please use `CLAccountHash` for identifier instead of `CLPublicKey`.